cron/polygon/standalone-runner.js

```diff
@@ -153,18 +154,48 @@ async function roughQuoteXInMATIC(xAmount, xAddress, xMATICLPPair) {
 }
 
 // determines the gas price by taking the minimum of "locally set max gas" and the gas price returned from api
-async function getGasPrice() {
-  return 65000000000 // 65 gwei in Polygon
+async function getFeeData() {
+  const gasPriceMin = 65e9 // 65 gwei minimum in Polygon
+  const maxPriorityFeePerGas = 2.51e9
```

Comment:
@farmer69420 I'm not sure this is enough on Polygon. In my recent experience the priority fee needed to get transactions through has been fluctuating a lot and can be as high as 50 GWEI at times.